lib/galaxy/model/__init__.py

```diff
@@ -805,6 +805,8 @@ class User(Base, Dictifiable, RepresentById):
         back_populates="user", order_by=lambda: desc(UserAddress.update_time), cascade_backrefs=False
     )
     custos_auth: Mapped[List["CustosAuthnzToken"]] = relationship(back_populates="user")
+    # TODO: Fixme, this rebase predates sqlalchemy2.0 and needs an update.
```

Comment:
```suggestion
```
looks fine!

Reply:
Hah, I fixed it up but left the old comment in.  I'll rebase it out after tests pass.